### PR TITLE
[script.video.nfl.gamepass] 2023.07.05

### PR DESCRIPTION
**Pull request by jm-duke**
### Description
**Addon**

Name: NFL Game Pass
ID: script.video.nfl.gamepass
Version: n/a
Source: https://github.com/pigskin/kodi-gamepass

**Info**

NFL has canceled the NFL Gamepass service, making this add-on obsolete. This PR updates the add-on to the latest code base and marks it as broken.

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

### CODE REVIEW

**Comment by romanvm** [comment]
Please mark the addon as broken instead of removing it: https://kodi.wiki/view/Addon.xml#.3Clifecyclestate.3E

**Comment by jm-duke** [comment]
Force pushed an update, marking the add-on as broken as requested.

File: script.video.nfl.gamepass/addon.xml [hunk 1]
```diff
@@ -15,17 +15,17 @@
   <extension point="xbmc.addon.metadata">
     <summary lang="en_GB">NFL Game Pass</summary>
     <description lang="en_GB">Watch NFL Game Pass streams.</description>
+    <lifecyclestate>broken</lifecyclestate>
```
Comment (romanvm) [inline script.video.nfl.gamepass/addon.xml:18]: Please carefully read the docs for `<lifecyclestate>` tag and fix it.

Comment (jm-duke) [reply script.video.nfl.gamepass/addon.xml]: Sorry, I shouldn't code when distracted...

**Review by romanvm — APPROVED** [review]
LGTM since it disables the addon.

**Pull request merged**